miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebChromeClient.kt

```diff
@@ -163,4 +164,21 @@ internal class MiniAppWebChromeClient(
         if (customView != null)
             onHideCustomView()
     }
+
+    @Suppress("TooGenericExceptionCaught", "SwallowedException")
+    override fun onShowFileChooser(
+        webView: WebView?,
+        filePathCallback: ValueCallback<Array<Uri>>?,
+        fileChooserParams: FileChooserParams?
+    ): Boolean {
+        try {
+            miniAppFilePicker?.requestFile(filePathCallback) { requestCode ->
+                val intent = fileChooserParams?.createIntent()
+                (context as Activity).startActivityForResult(intent, requestCode)
+            }
+        } catch (e: Exception) {
+            throw (MiniAppFilePickingException(e.message.toString()))
```

Comment:
What happens with the Mini App/Host App when this exception is thrown? There's no place where the exception is caught, so isn't this going to crash the App?

Maybe we should just return an empty array to the mini app if there was an exception?

Reply:
`onShowFileChooser` is expecting a `Boolean`, so I am not sure if we can return an empty array, instead maybe we can do `Log.e` the error message? 🤔

Reply:
@khairul-alam-licon Yes, I think logging the error and then return `false` is the right solution. It's better than crashing the App 😆

Reply:
Haha, true 😄
I have updated this part accordingly...